WEB_SERVICE/WEB/_MEDIUM/GRAPHQL_DEBUG_MODE/description.md

```diff
@@ -0,0 +1,32 @@
+GraphQL Debug Mode is a feature that, when enabled, provides detailed error messages and stack traces in responses. While this is invaluable during development, leaving it enabled in a production environment can potentially expose sensitive information about the server's internal structure and implementation details.
+
+When debug mode is enabled, error responses may include:
+- Detailed stack traces
+- Database query information
+- Internal server paths and file names
+- Sensitive configuration details
+
+Example of a response with debug mode enabled:
+
+```json
+{
+  "errors": [
+    {
+      "message": "Cannot query field \"nonexistentField\" on type \"Query\".",
+      "locations": [
+        {
+          "line": 3,
+          "column": 3
+        }
+      ],
+      "stack": "GraphQLError: Cannot query field \"nonexistentField\" on type \"Query\".\n    at validateField (/app/node_modules/graphql/validation/rules/FieldsOnCorrectTypeRule.js:48:31)\n    at ..."
+    }
+  ],
+  "data": null
+}
+```
```

Comment:
Nah, this is not an example of debug mode. Please check `debug` in Django Graphene for an example where you can access the SQL queries running.